src/installer/Directory.Build.props

```diff
@@ -129,7 +130,7 @@
   <PropertyGroup>
     <DisableCrossgen>false</DisableCrossgen>
     <!-- Disable cross-gen on FreeBSD for now. This can be revisited when we have full support. -->
-    <DisableCrossgen Condition="'$(TargetOS)'=='FreeBSD'">true</DisableCrossgen>
+    <DisableCrossgen Condition="'$(TargetOS)'=='FreeBSD' or '$(RuntimeFlavor)' == 'Mono'">true</DisableCrossgen>
```

Comment:
Put on separate line to simplify FreeBSD fix later, and clarify the target of the comment?